pkg/config/engine/containerd/option.go

```diff
@@ -78,3 +78,14 @@ func WithContainerAnnotations(containerAnnotations ...string) Option {
 		b.containerAnnotations = containerAnnotations
 	}
 }
+
+// CommandLineSource returns the CLI-based containerd config loader
+func CommandLineSource(hostRoot string) toml.Loader {
```

Comment:
This shouldn't be in the `option.go` file.

We should also use this for the `(hostRoot == "" || hostRoot == "/")` case.

Reply:
How about this?

```
// CommandLineSource returns the CLI-based containerd config loader
func CommandLineSource(hostRoot string) toml.Loader {
	var cliArgs []string
	if hostRoot != "" && hostRoot != "/" {
		cliArgs = append(cliArgs, "chroot", hostRoot)
	}
	cliArgs = append(cliArgs, "containerd", "config", "dump")

	return toml.FromCommandLine(cliArgs[0], cliArgs[1:]...)
}
```